src/Reflection/ReflectionClass.php

```diff
@@ -160,7 +162,22 @@ public function getShortName(): string
             $fileName = sha1($this->locatedSource->getSource());
         }
 
-        return sprintf('%s%c%s(%d)', self::ANONYMOUS_CLASS_NAME_PREFIX, "\0", $fileName, $this->getStartLine());
+        return sprintf('%s%s%c%s(%d)', $this->getAnonymousClassNamePrefix(), self::ANONYMOUS_CLASS_NAME_SUFFIX, "\0", $fileName, $this->getStartLine());
+    }
+
+    private function getAnonymousClassNamePrefix(): string
+    {
+        $parentClassNames = $this->getParentClassNames();
```

Comment:
`getParentClassNames` and `getInterfaceNames` need to return a `list` here - can we improve that?

Reply:
It's already done.

https://github.com/Roave/BetterReflection/blob/5.0.x/src/Reflection/ReflectionClass.php#L862
https://github.com/Roave/BetterReflection/blob/5.0.x/src/Reflection/ReflectionClass.php#L1165